test/bdd/pkg/issuer/issuer_steps.go

```diff
@@ -250,12 +264,34 @@ func (e *Steps) retrieveProfile(id, name, issuerURL, supportedVCContexts,
 	return nil
 }
 
-func (e *Steps) walletConnect(issuerID string) error {
+func (e *Steps) walletConnect(profileID string) error {
 	state := uuid.New().String()
-	e.states[issuerID] = state
+	e.states[profileID] = state
+
+	resp, err := bddutil.HTTPDo(http.MethodGet, //nolint: bodyclose
+		fmt.Sprintf(AdapterURL+"/%s/connect/wallet?state=%s", profileID, state), "", "", nil,
+		e.bddContext.TLSConfig())
+	if err != nil {
+		return fmt.Errorf("failed to execute wallet request: %w", err)
+	}
+
+	defer bddutil.CloseResponseBody(resp.Body)
+
+	// validating only status code as the vue page needs javascript support
```

Comment:
Not sure what vue pages/Javascript have to do with this. Can you explain?

(It might just be me who isn't familiar with how this works, so let me know)

Reply:
This is the existing code i only changed variable name and this endpoint gets automatically redirects to vue page thats why its written here that we are only testing status code